### PR TITLE
switch the metatype attribute for infinispan support to beta

### PR DESCRIPTION
**Pull request by njr-11**
Mark the metatype attribute for gating infinispan support as beta, and add a translatable name/description.

### CODE REVIEW

File: dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -30,6 +30,9 @@ properties.desc=List of vendor specific JCache configuration properties, which a
 extraProperties.name=Vendor specific properties
 extraProperties.description=List of vendor specific JCache configuration properties, which are passed to the JCache provider when obtaining the CacheManager.
 
+enableBetaSupportForInfinispan.name=Enable beta of support for Infinispan
+enableBetaSupportForInfinispan.desc=Enables Infinispan to be used for HTTP session persistence as beta function.
```
[the remark this answers is not in the document]
Comment (njr-11) [reply dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties]: The update that you requested has been made under https://github.com/OpenLiberty/open-liberty/pull/9556/commits/e4f3ef943dc5cb23aceddd6b4f59ad8703ddb5d2

[the remark this answers is not in the document]
Comment (ManasiGandhi) [reply dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties]: I made a small addition.

[the remark this answers is not in the document]
Comment (njr-11) [reply dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties]: Where can I view the small addition that you referred to? I don't see a comment or an additional commit for it.  I'm sure it's fine.  I just don't know where to look for it.

**Comment by ManasiGandhi** [comment]
You can view them here. I also had a question before you build. Is
Infinispan, in this case, a server?

https://github.com/OpenLiberty/open-liberty/pull/9539#discussion_r340751867

**Comment by ManasiGandhi** [comment]
sorry sent you the wrong link

File: dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (ManasiGandhi) [reply dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties]: You can view it here https://github.com/OpenLiberty/open-liberty/pull/9556#discussion_r340644868

**Comment by skasund** [comment]
L2 message review not required

**Pull request merged**